arm-notificationhubs/2016-03-01/swagger/notificationhubs.json

```diff
@@ -129,6 +129,52 @@
           }
         }
       },
+	  "patch": {
+		"tags": [
+          "Namespaces"
+        ],
+		"operationId": "Namespaces_Patch",
+		"description": "Patches the existing namespace",
+		"parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
```

Comment:
@anudeepsharma This [extension](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-extensions.md#x-ms-parameter-location) should provide you more info.